c/include/libsbp/system.h

```diff
@@ -82,12 +82,38 @@ typedef struct SBP_ATTR_PACKED {
  * The INS status message describes the state of the operation
  * and initialization of the inertial navigation system. 
  */
-#define SBP_MSG_INS_STATUS   0xFF03
+#define SBP_MSG_INS_STATUS            0xFF03
 typedef struct SBP_ATTR_PACKED {
   u32 flags;    /**< Status flags */
 } msg_ins_status_t;
 
 
+/** Experimental telemetry message
+ *
+ * The CSAC telemetry message has an implementation defined telemetry string
+ * from a device. It is not produced or available on general Swift Products.
+ * It is intended to be a low rate message for status purposes.
+ */
+#define SBP_MSG_CSAC_TELEMETRY        0xFF04
+typedef struct SBP_ATTR_PACKED {
+  u8 id;           /**< Index representing the type of telemetry in use.  It is implemention defined. */
+  char telemetry[0]; /**< Comma separated list of values as defined by the index */
+} msg_csac_telemetry_t;
+
+
+/** Experimental telemetry message labels
+ *
+ * The CSAC telemetry message provides labels for each member of the string
+ * produced by MSG_CSAC_TELEMETRY. It should be provided by a device at a lower
+ * rate than the MSG_CSAC_TELEMETRY.
+ */
+#define SBP_MSG_CSAC_TELEMETRY_LABELS 0xFF05
```

Comment:
Do we need a message for the labels? Don't we already know what data comes out in the telemetry? If it ever changes, we could just rev the `SBP_MSG_CSAC_TELEMETRY` message.

Reply:
The labels come from the CSAC.